js/packages/navigation/stacks.tsx

```diff
@@ -144,19 +143,6 @@ export const CreateGroupNavigation: React.FC = () => {
 	)
 }
 
-const TabStack = createMaterialTopTabNavigator() // provides swipe animation
-export const TabNavigation: React.FC = () => {
-	return (
-		<TabStack.Navigator
-			initialRouteName={Routes.Main.Home}
-			tabBar={() => <Components.Main.Footer />}
-			tabBarPosition='bottom'
-		>
-			<TabStack.Screen name={Routes.Main.Home} component={Components.Main.Home} />
-		</TabStack.Navigator>
-	)
-}
```

Comment:
In this navigator you can see that there is a props `tabBar` and it's associate to our footer component.
We have kept that materialBar because we just change UI/UX, and before the navigator included more Screens that had this footer
We can remove it, we have a swipeProvider that based on this materialBar and normally it's not use, so you can remove it too if you remove this part, it's `SwipeHelperReactNavTabBar` in `shared-components/SwipeNavRecogniser.tsx`

Reply:
Okay.
 I have already added Footer directly to `main/Home.tsx`
I have updated this PR removing `SwipeHelperReactNavTabBar` from `shared-components/SwipeNavRecogniser.tsx`